ykllvmwrap/src/jitmodbuilder.cc

```diff
@@ -461,20 +550,14 @@ class JITModBuilder {
       assert(MaybeIB.hasValue());
       IRBlock IB = MaybeIB.getValue();
 
-      // Get a traced function so we can extract blocks from it.
-      Function *F = AOTMod->getFunction(IB.FuncName);
-      if (!F)
-        errx(EXIT_FAILURE, "can't find function %s", IB.FuncName);
+      Function *F;
+      BasicBlock *BB;
+      std::tie(F, BB) = getLLVMAOTFuncAndBlock(&IB);
```

Comment:
What does `std::tie` do here?

Reply:
it unpacks a tuple.

(and no, you can't put the type inside the tie, I tried).

This was made more concise in c++17, but we don't target it, I don't think.

Reply:
Got it. [I have no preferences on C++ versions. I'm fine with whatever.]

Reply:
Alas:
```
  cargo:warning=src/jitmodbuilder.cc:552:12: error: decomposition declarations are a C++17 extension [-Werror,-Wc++17-extensions]
  cargo:warning=      auto [F, BB] = getLLVMAOTFuncAndBlock(&IB);
  cargo:warning=           ^~~~~~~
  cargo:warning=1 error generated.
  exit status: 1
```

I don't fancy changing the C++ standard, so let's just roll with it.

Reply:
OK.